TLSharp.Core/TelegramClient.cs

```diff
@@ -377,8 +377,15 @@ private void OnUserAuthenticated(TLUser TLUser)
             _session.SessionExpires = int.MaxValue;
 
             _session.Save();
-        }
-
+        }
+
+        public bool IsConnected()

```

Comment:
Resource demanding? It's only checking for null and then later returning a property.